src/content/docs/kubernetes-pixie/kubernetes-integration/understand-use-data/kubernetes-cluster-explorer.mdx

```diff
@@ -267,8 +267,15 @@ If you've enabled the [Kubernetes events integration](/docs/integrations/kuberne
 
 ## Use the cluster explorer [#cluster-explorer-use]
 
-<Callout title="Deprecated">
-  The new Kubernetes Navigator UI is intended to replace the cluster explorer. However, you can enable the original cluster explorer experience at any time by turning off the Navigator toggle switch.
+<Callout
+  variant="important"
+  title="EOL NOTICE"
+>
+
+  The new Kubernetes Navigator UI is intended to replace the cluster explorer. However, you can enable the original Cluster Explorer experience at any time by turning off the Navigator toggle switch.
```

Comment:
```suggestion
  The new Kubernetes Navigator UI is intended to replace the cluster explorer. However, you can enable the original cluster explorer experience at any time by turning off the navigator toggle switch.
```

Reply:
Where is the navigator toggle switch? Consider adding more context or a screenshot

Reply:
There is a screenshot right below the note. Do you think I should include the screenshot in the note?